editorconfig/versiontools.py

```diff
@@ -6,23 +6,25 @@
 """
 
 import re
+from typing import Optional, Tuple
```

Comment:
`Optional` is still needed on 3.9, but `Tuple` can be written as `tuple`.

Reply:
You're absolutely right, I was targeting 3.8 when I wrote it, but with 3.9, it's unnecessary . I'll update all Tuples to tuple and Lists to list.